test/OpenTelemetry.Instrumentation.AWSLambda.Tests/AWSLambdaWrapperTests.cs

```diff
@@ -261,8 +261,13 @@ private void AssertSpanProperties(Activity activity, string parentId)
         Assert.Matches(@"^\d+(\.\d+){3}$", activity.Source.Version);
     }
 
-    private void AssertResourceAttributes(Resource resource)
+    private void AssertResourceAttributes(Resource? resource)
     {
+        if (resource == null)
```

Comment:
I don't think a null resource should pass the assertion.

Reply:
Added null assertion for the `resource `